src/media/js/site/components/pageHeader.js

```diff
@@ -0,0 +1,16 @@
+import classnames from 'classnames';
+import {connect} from 'react-redux';
```

Comment:
i find it's better to order by the latter section since the former section is hard to order alphabetically due to named imports.

```js
'classnames'
'react'
'react-redux'
'react-router-reverse'
```